spring-cloud-commons/src/test/java/org/springframework/cloud/client/loadbalancer/reactive/ReactorLoadBalancerExchangeFilterFunctionTests.java

```diff
@@ -50,7 +50,7 @@
 import org.springframework.cloud.client.loadbalancer.Response;
 import org.springframework.context.annotation.Bean;
 import org.springframework.http.HttpStatus;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
```

Comment:
The new import is placed according to the custom import order styling applied in Rewrite's configuration.